src/language/en-GB/en-GB.plg_api_reports.sys.ini

```diff
@@ -0,0 +1 @@
+PLG_API_REPORTS = "TJ Reports Api Plugin"
```

Comment:
@monikaatvowel either use PLG_REPORTS_API this or PLG_API_REPORTS this as a prefix

Reply:
use `PLG_API_REPORTS`

Reply:
done